Neos.Flow/Tests/Unit/Cli/ConsoleOutputTest.php

```diff
@@ -0,0 +1,299 @@
+<?php
+namespace Neos\Flow\Tests\Unit\Cli;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Cli;
+use Neos\Flow\Cli\ConsoleOutput;
+use Neos\Flow\Tests\UnitTestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\StreamOutput;
+
+/**
+ * Test cases for CLI console output helpers
+ */
+class ConsoleOutputTest extends UnitTestCase
```

Comment:
Great to have tests for these methods. As they are testing external state (php://memory) I would classify them as functional tests though.

Reply:
I disagree. They test a single unit, it's just that the result isn't returned directly. Our functional tests fire up a full framework, more or less, which isn't the case here.